src/main/plugin/iso19139.ca.HNAP/loc/eng/codelists.xml

```diff
@@ -2023,14 +2023,14 @@
 			<description>Information applies to a computer program or routine</description>
 		</entry>
 		<!-- - - - - - - - - - - - - - - - - - - - - - - - - -->
-		<entry>
```

Comment:
Why do you need this change?
Won't  this be fixed in the followings lines that you added?

```
<codelist name="gmd:MD_ScopeCode"
            displayIf="/ancestor::node()[name()='gmd:MD_Metadata']/gmd:identificationInfo/srv:SV_ServiceIdentification">
```

Reply:
This change is needed. There will be two code list as what Jose's comment. I am adding one for xPath displayIf="/ancestor::node()[name()='gmd:MD_Metadata']/gmd:identificationInfo/srv:SV_ServiceIdentification"

So the original service needs to be removed from the previous original code list which is not for service type record. I have confirmed from catalogue editor mode and it's really needed to be removed. Otherwise if it remains, the dataset type of record will still have the option to switch to service which will break the validation.